2009-01-21  Aurelien Jarno  <aurelien@aurel32.net>

	* linuxthreads/descr.h (thread_self): Remove static to prevent
	  warning.

---
 linuxthreads/descr.h |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/linuxthreads/descr.h
+++ b/linuxthreads/descr.h
@@ -260,8 +260,8 @@
 
 extern pthread_descr __pthread_find_self (void) __attribute__ ((pure));
 
-static inline pthread_descr thread_self (void) __attribute__ ((pure));
-static inline pthread_descr thread_self (void)
+extern inline pthread_descr thread_self (void) __attribute__ ((pure));
+extern inline pthread_descr thread_self (void)
 {
 #ifdef THREAD_SELF
   return THREAD_SELF;
